config/software/datadog-process-agent.rb

```diff
@@ -0,0 +1,15 @@
+name "datadog-process-agent"
+always_build true
+
+build do
+  agent_branch = ENV['PROCESS_AGENT_BRANCH']
```

Comment:
Also, I actually skimmed over this way too quickly earlier. I would probably move this out of the `build` block, and set the agent_branch there. As well as setting the `default_version` - I don't think it's necessary, but it'll help be consistent with other definitions.